Makefile

```diff
@@ -219,8 +219,7 @@ test.integration.legacy: container
 test.e2e:
 	GOFLAGS="-tags=e2e_tests" go test -v \
 		-race \
-		-timeout 15m \
-		-parallel $(NCPU) \
+		-timeout 45m \
```

Comment:
Actually that was an overshot. I've seen the test take 8 minutes, and also 16. I'll do 20 for now to give it some breathing room.

```suggestion
		-timeout 20m \
```

Reply:
Saw a run where Github Actions did about 17m before finishing, so I think maybe 20m is cutting it too low. I bumped to 30m, the problem is that Github Actions appears to be somewhat unpredictable in how long it will take for it to run this test.